python/pyproject.toml

```diff
@@ -63,7 +63,7 @@ autogen = [
     "autogen-agentchat >= 0.2, <0.4"
 ]
 aws = [
-    "boto3>=1.36.4,<1.38.0",
+    "boto3>=1.36.4,<1.39.0",
```

Comment:
This change is related to a dependabot PR that tried to do the same, but ran into conflicts. Decided to handle it here.